package.json

```diff
@@ -1023,18 +1023,7 @@
     "onDebugResolve:clr",
     "onDebugResolve:dotnet",
     "onLanguage:csharp",
-    "onLanguage:aspnetcorerazor",
```

Comment:
updating the version added a warning here that these were not necessary (they are automatically generated now based on the commands we list).

Reply:
Why remove the "onLanguage:aspnetcorerazor" one? was it similarly not necessary?

Reply:
yup!
![image](https://github.com/dotnet/vscode-csharp/assets/5749229/597f4e5c-3bd8-455f-bb6e-0dc0109a6d40)